packages/api-v2/src/hooks/useTwoFactorAuthenticationStatus.ts

```diff
@@ -0,0 +1,18 @@
+import useQuery from '../useQuery';
+import useAuthorize from './useAuthorize';
+
+/** A custom hook to get the two factor authentication status. i.e. Whether it's enabled or not */
+const useTwoFactorAuthenticationStatus = () => {
```

Comment:
just a nitpick:
wouldn't it be better to rename the hook to something like useIsTwoFactorAuthenticationEnabled,
it would be more understandable form the name itself, that it returns us boolean value, and not exact 'status'